nightly/nightly.txt

```diff
@@ -146,6 +146,9 @@ expensive nearcore test_cases_testnet_rpc test::test_add_access_key_with_allowan
 expensive nearcore test_cases_testnet_rpc test::test_delete_access_key_with_allowance_testnet
 expensive nearcore test_cases_testnet_rpc test::test_access_key_smart_contract_testnet
 expensive nearcore test_cases_testnet_rpc test::test_evm_deploy_call_testnet
+expensive nearcore test_cases_testnet_rpc test::test_evm_fibonacci_gas_limit_testnet
```

Comment:
this file is deprecated. Please add the tests to nightly/tests_for_nayduck.txt. Also are these tests actually very expensive to run?

Reply:
No, but sanity CI complains

Reply:
See https://buildkite.com/nearprotocol/nearcore/builds/5054#b90c35a4-41c6-47ff-b876-9d6f1dad41ae